internal/proto/daemon.proto

```diff
@@ -65,9 +64,16 @@ message Output {
 //     repeated ServiceStatus statuses = 1;
 // }
 
+enum LogLevel {
+    INFO = 0;
+    VERBOSE = 1;
+    DEBUG = 2;
+}
+
 message CreateServiceRequest {
     Service service = 1;
     bool watch = 2;
+    LogLevel log_level = 3;
```

Comment:
i think it's important to be specific of who the expected audience of this log level is. How do I decide which level to use? how do i decide whether to send progress information to the log vs. the CreateServiceReply? there should be with written comments documenting the decision

Reply:
re: log vs CreateServiceReply - my plan is to hook the logger up to the CreateServiceReply. maybe the name "logger" is misleading if this is what we're using to report output to the user?

I've added comments to the `Logger` interface, which might be the least bad place to put them, but open to thoughts.